sql/01-bde_roles.sql

```diff
@@ -14,6 +14,36 @@
 DO $ROLES$
 BEGIN
 
+-- bde_user
+
+IF NOT EXISTS (SELECT * FROM pg_roles where rolname = 'bde_user') THEN
+    CREATE ROLE bde_user
+        NOSUPERUSER INHERIT NOCREATEDB NOCREATEROLE;
+    ALTER ROLE bde_user SET search_path=bde, public;
+END IF;
+COMMENT ON ROLE bde_user IS $COMMENT$
+Has rights to execute and read anything that bde_dba creates.
+
+Used by users logging in to run analytics queries or for remote users
+to get access to data (e.g LDS import or QGIS desktop users)
+$COMMENT$;
+
+-- bde_admin
+
+IF NOT EXISTS (SELECT * FROM pg_roles where rolname = 'bde_admin') THEN
+    CREATE ROLE bde_admin
+        NOSUPERUSER INHERIT NOCREATEDB NOCREATEROLE;
+    ALTER ROLE bde_admin SET search_path=bde, public;
+END IF;
+
+GRANT bde_user TO bde_admin;
+
+COMMENT ON ROLE bde_admin IS $COMMENT$
+Everything that user can do plus the ability to edit data in the tables.
```

Comment:
Would you please update this role comment to something similar than we have in `bde_user` role - like `Has rights to ...`